dm/master/openapi.go

```diff
@@ -215,12 +221,96 @@ func (s *Server) DMAPIGetSourceStatus(ctx echo.Context, sourceName string) error
 
 // DMAPIStartTask url is:(POST /api/v1/tasks).
 func (s *Server) DMAPIStartTask(ctx echo.Context) error {
-	return nil
+	var req openapi.CreateTaskRequest
+	if err := ctx.Bind(&req); err != nil {
+		return err
```

Comment:
```suggestion
		return errors.Trace(err)
```

Reply:
We should wrap error produced by other module

Reply:
added in https://github.com/pingcap/dm/pull/2141/commits/7f475c757fb9769eae7f19c7726222968fe931aa